src/main/java/com/targus/ui/WSNPrototype.java

```diff
@@ -0,0 +1,196 @@
+package com.targus.ui;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.targus.utils.Constants;
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+import javafx.geometry.Point2D;
+import javafx.stage.FileChooser;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class WSNPrototype {
+    private int paneWidth;
+    private int paneHeight;
+    private final ArrayList<Point2D> potentialPositions = new ArrayList<Point2D>();
+
+    public ArrayList<Point2D> getPotentialPositions() {
```

Comment:
The class has a default constructor and initially, all class members will contain `null` or empty values.  Can we add another return value or exception for `null` and empty values?